packages/kit/types/app.d.ts

```diff
@@ -1,9 +1,15 @@
 import { ReadOnlyFormData, RequestHeaders } from './helper';
 import { ServerResponse } from './hooks';
+import { PrerenderOptions, SSRManifest } from './internal';
```

Comment:
we probably shouldn't import things from `internal`. if they're being exposed they should be moved here

Reply:
done